src/resources/container.js

```diff
@@ -175,7 +175,8 @@ class ContainerResource {
             if (attachedMi) {
                 entity.addComponent("render", Object.assign({
                     type: "asset",
-                    meshInstances: attachedMi
+                    meshInstances: attachedMi,
+                    rootBone: root
```

Comment:
and this line can be perhaps removed now as well?

Reply:
From my testing it looks like the rootBone has to be set on the entity you're cloning, otherwise the skin of the clone breaks.

Reply:
ok that seems ok .. look at SkinInstanceCache.createCachedSkinedInstance which is called 10 line bellow, we effectively use root as a rootBone as well.